### DISCUSSION TITLE
automatic addition of an event

### DISCUSSION THREAD

**Discussion by Mishanki** [spatie/icalendar-generator#83] [Bugs]
Can I **automatic** addition of an event from email invite.ics file to outlook or iCalendar?
<img width="361" alt="Screenshot 2022-03-03 at 16 42 55" src="https://user-images.githubusercontent.com/11581729/156576585-1e372043-2de1-4174-b753-776fa228b718.png">

**Comment by rubenvanassche** [comment]
Hi @Mishanki,

The Ical standard does not describe such thing, so I don't think it is possible. It depends on the calendar/email client.
